config/routes.rb

```diff
@@ -26,6 +26,9 @@
 
     patch '/confirm' => 'users/confirmations#confirm'
 
+    get '/users/totp' => 'users/totp_setup#index'
+    get '/users/totp_disable' => 'users/totp_setup#disable', as: :disable_totp
```

Comment:
In general, it's not recommended to allow destructive behavior via GET. Those actions should only be accessible via POST or DELETE requests. To adhere to REST and CRUD conventions, I would make this a DELETE request that points to a method called  `destroy`.